src/libertem_holo/base/reconstr.py

```diff
@@ -109,10 +109,10 @@ def estimate_sideband_position(holo_data, holo_sampling, central_band_mask_radiu
     # Sideband position in pixels referred to unshifted FFT
     if sb == 'lower':
         fft_sb = fft_filtered[:int(fft_filtered.shape[0] / 2), :]
-        sb_position = np.asarray(np.unravel_index(fft_sb.argmax(), fft_sb.shape))
+        sb_position = np.asarray(np.unravel_index(np.abs(fft_sb).argmax(), fft_sb.shape))
     elif sb == 'upper':
         fft_sb = fft_filtered[int(fft_filtered.shape[0] / 2):, :]
-        sb_position = (np.unravel_index(fft_sb.argmax(), fft_sb.shape))
+        sb_position = (np.unravel_index(np.abs(fft_sb).argmax(), fft_sb.shape))
```

Comment:
Can you explain these changes? Is this necessary because of the definition of `argmax` on complex numbers? I think that by default operates on the real part, right?

Reply:
I try it in my notebook and find with the np.abs, it is more accurate. Moreover I just find this github page, that suggest the np.argmax does not behave correctly without np.abs (https://github.com/numpy/numpy/issues/10469).